debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat

```diff
@@ -34,6 +34,8 @@
 
 CLimits.USHRT_MAX = 0xFFFF
 
+J9BuildFlags.gc_enableSparseHeapAllocation = 0
```

Comment:
it's unclear if this will cause problems if no other changes are brought before (like OMR introducing this flag)

Reply:
likely path is to introduce the flag in OMR first, but have it OFF